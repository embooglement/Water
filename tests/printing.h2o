print(0);
print(1, 2, 3);
println();
println("hello world");
