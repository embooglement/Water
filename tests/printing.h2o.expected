01 2 3
hello world

